### PR TITLE
Fix links to past content

### PR DESCRIPTION
**Pull request by scottyhq**
small fix to content links

### CODE REVIEW

**Comment by JessicaS11** [comment]
Had just noticed this and see you're one step ahead of me. @scottyhq can you add the preview label?

**Comment by scottyhq** [comment]
> Had just noticed this and see you're one step ahead of me. @scottyhq can you add the preview label? 

Do you not have permissions to do that? I'm still working out how to more easily manage permissions in all these various orgs...

File: cookiecutter.yaml [hunk 1]
```diff
@@ -16,10 +16,10 @@ banner:
   location: Seattle, WA
   links:
   - url: intro.html
-    title: Event Jupyter Book
+    title: 2022 Jupyter Book
     new_window: false
-  - url: https://snowex.hackweek.io/intro.html
-    title: 2022 Website
+  - url: https://snowex-hackweek.github.io/website/tutorials/index.html
```
Comment (JessicaS11) [inline cookiecutter.yaml]: Do you want it to go directly to the tutorials page?

**Comment by JessicaS11** [comment]
> Do you not have permissions to do that? 

Nope - I'm not an admin in this org.

File: cookiecutter.yaml (hunk 1 above)
Comment (scottyhq) [reply cookiecutter.yaml]: fixed

**Comment by scottyhq** [comment]
> I'm not an admin in this org. 

Looks like the 'admin' team didn't have access to this repository... should be fixed now

**Review by JessicaS11 — APPROVED** [review]

**Pull request merged**